src/components/pages/Triggers/TriggerDetails/TriggerSettings/TriggerCondition/ResourceCondition.tsx

```diff
@@ -0,0 +1,162 @@
+import {DeleteOutlined} from '@ant-design/icons';
+import {Form, Input, InputNumber, Select} from 'antd';
+
+import {Button, FormItem, FormRow, FullWidthSpace} from '@custom-antd';
+import {SymbolWrapper} from '@custom-antd/Form/Form.styled';
```

Comment:
I'm not sure what it actually does, the name doesn't say anything meaningful (given that some `Delete` button is used here as a child);

* Is the name correct?
* Should it be part of `@custom-antd/Form`, or maybe it's not connected?
* Should it be generic, or rather local for `ResourceCondition`?